From e4936e9197a926ff7fd0a75598d36b9468519ef9 Mon Sep 17 00:00:00 2001
From: "Wu, Hao" <hao.wu@intel.com>
Date: Tue, 3 Sep 2013 22:58:12 +0800
Subject: [PATCH 227/429] dwc_otg: add SDP case for power current

Add SDP case, othwise it will return error which blocks
the main thread handling.

Signed-off-by: Wu, Hao <hao.wu@intel.com>
Signed-off-by: Wang, Yu <yu.y.wang@intel.com>
---
 drivers/usb/dwc3/otg.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/dwc3/otg.c b/drivers/usb/dwc3/otg.c
index 06124b6f589b..7141e7c0779d 100644
--- a/drivers/usb/dwc3/otg.c
+++ b/drivers/usb/dwc3/otg.c
@@ -455,6 +455,8 @@ static enum dwc_otg_state do_charger_detection(struct dwc_otg2 *otg)
 	case POWER_SUPPLY_CHARGER_TYPE_SE1:
 		ma = 1500;
 		break;
+	case POWER_SUPPLY_CHARGER_TYPE_USB_SDP:
+		break;
 	default:
 		otg_err(otg, "Charger type is not valid to notify battery\n");
 		return -EINVAL;
-- 
2.37.3

